packages/web/src/components/tabbedPane.tsx

```diff
@@ -82,9 +82,9 @@ export const TabbedPane: React.FunctionComponent<{
         tabs.map(tab => {
           const className = 'tab-content tab-' + tab.id;
           if (tab.component)
-            return <div key={tab.id} className={className} style={{ display: selectedTab === tab.id ? 'inherit' : 'none' }}>{tab.component}</div>;
+            return <div key={tab.id} id={`tab-${tab.id}`} role='tabpanel' title={tab.title} className={className} style={{ display: selectedTab === tab.id ? 'inherit' : 'none' }}>{tab.component}</div>;
```

Comment:
- I am not sure you'd like the whole tabpanel to have this title. I'd expect titles on internal elements of the tab content, not a single one covering the whole panel.
- I am afraid that `id`s will clash between various tabbed pane. Should we prefix them with some unique value for this `TabbedPane`?